DuckDuckGo/UnifiedFeedbackForm/UnifiedFeedbackFormViewModel.swift

```diff
@@ -236,3 +306,10 @@ final class UnifiedFeedbackFormViewModel: ObservableObject {
         }()
     }
 }
+
+private extension String {
+    var isValidEmail: Bool {
+        guard let regex = try? NSRegularExpression(pattern: #"[^\s]+@[^\s]+\.[^\s]+"#) else { return false }
```

Comment:
Same as the one used in https://github.com/duckduckgo/BrowserServicesKit/blob/3e3e700d888f23c9b22abbcede78cd53612104b2/Sources/Common/Extensions/StringExtension.swift#L33